lib/tracing-policy.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function RateLimiterPolicy(samplesPerSecond) {
+  if (samplesPerSecond > 1000) {
+    samplesPerSecond = 1000;
+  } else if (samplesPerSecond === 0) {
```

Comment:
If the user says 0 samples per second, I think it is incorrect for us to take silently samples. It would be better to throw.